tools/pythonpkg/tests/extensions/test_httpfs.py

```diff
@@ -70,3 +70,16 @@ def test_http_exception(self, require):
         assert value.status_code != 200
         assert value.body == ''
         assert 'Content-Length' in value.headers
+
+    def test_fsspec_priority(self, require):
+        pytest.importorskip("fsspec")
+        pytest.importorskip("gscfs")
```

Comment:
I think this will not be tested because the python extension CI job will not have `gscfs` installed, do we want to add this to make sure it gets tested on CI?

Reply:
Instead we could create a dummy filesystem that corresponds to the "gcs" protocol, just to test that it gets picked over the system one

Reply:
`tools/pythonpkg/tests/fast/test_filesystem.py` does this I believe, in `test_multiple_protocol_filesystems`